src/components/Auth/Auth.jsx

```diff
@@ -16,24 +16,23 @@ export default class Auth extends React.Component {
         return status < 500;
       }})
       .then(res => {            
-          if (res.status === 200) {
```

Comment:
I think we're using 4 spaces tabs everywhere else, how come you've changed this file to 2?

Reply:
Oh damn, it was when I copied it over from slack ahhah, sorry